support/mock/builder.go

```diff
@@ -13,16 +13,25 @@ import (
 
 // Build for fluent initialization of a mock runtime.
 type RuntimeBuilder struct {
-	rt *Runtime
+	options []func(rt *Runtime)
 }
 
 // Initializes a new builder with a receiving actor address.
-func NewBuilder(ctx context.Context, receiver addr.Address) *RuntimeBuilder {
+func NewBuilder(ctx context.Context, receiver addr.Address) RuntimeBuilder {
+	var b RuntimeBuilder
+	b.add(func(rt *Runtime) {
+		rt.ctx = ctx
+		rt.receiver = receiver
+	})
+	return b
+}
+
+// Builds a new runtime object with the configured values.
+func (b RuntimeBuilder) Build(t testing.TB) *Runtime {
 	m := &Runtime{
-		ctx:               ctx,
+		ctx:               context.Background(),
```

Comment:
This makes sense given the way the previous version behaved, but adding ctx as a parameter to `Build` could stave off some test problems in the future if this context is ever used.

Reply:
Hm. I agree. That's the right place to put the context.

Reply:
Actually, I'm just going to make this construct the context internally and return a cancel function. That's the "least easy to mess up" solution.

Reply:
That also avoids passing a confusing context. If I pass it as a parameter, it _looks_ like canceling the context should cancel the build operation.

Reply:
Actually, I just dropped it all together. Instead, I cancel the context when the test ends.